packages/manager/modules/web-paas/src/terminate/terminate.module.js

```diff
@@ -0,0 +1,14 @@
+import angular from 'angular';
+
+import component from '../details/service/terminate/terminate.component';
+import routing from './terminate.routing';
+
+const moduleName = 'ovhManagerWebPaasTerminate';
+
+angular
+  .module(moduleName, [])
+  .config(routing)
+  .component('webPaasTerminate', component)
+  .run(/* @ngTranslationsInject:json ../details/service/terminate/translations */);
```

Comment:
```suggestion
  .run(/* @ngTranslationsInject:json ./translations */);
```
New module should have their own translations

Reply:
This is a redirect module which has shared same translations with another module. Both share same view and controller but different routes/url. Hence we are sharing translations here so as to not repeat it.

Reply:
Then the translations should be imported by the module exposing the template/controller/component not by the modules using it and declaring the routing